vscode-workspaces/multiroot-go.code-workspace

```diff
@@ -41,9 +41,9 @@
 			"name": "src"
 		},
 		{
-			"path": "../wifi/cloud/go",
-			"name": "wifi/cloud",
-		},
+			"path": "../feg/gateway",
+			"name": "feg/gateway"
+		}
```

Comment:
why has this changed

Reply:
when I'm trying to open this workspace file I see message that there is no `wifi/cloud` destination, and in `feg/gateway` there is go code that we're working with